### PR TITLE
env var override for CACHE_EXPIRY_IN_SECS

### PR DESCRIPTION
**Pull request by harshkothari410**
- env var override for CACHE_EXPIRY_IN_SECS
- invalidate the cache on PUT/PATCH operation on Perspective
- Added test to verify above behavior

### CODE REVIEW

File: api/v1/controllers/perspectives.js [hunk 1]
```diff
@@ -160,6 +161,11 @@ module.exports = {
    */
   patchPerspective(req, res, next) {
     doPatch(req, res, next, helper);
+
+    // Remove from cache after successfull PATCH
```
Comment (iamigo) [inline api/v1/controllers/perspectives.js]: successful has only one "l"

File: tests/enableCache/perspectives.js [hunk 2]
```diff
@@ -100,6 +100,70 @@ describe(`tests/enableCache/perspectives.js, api: GET ${path} >`, () => {
...
+
+        redisCache.get('/v1/perspectives', (cacheErr, reply) => {
+          expect(reply).to.not.exist;
+          return done();
+        });
+      });
+    });
+  });
+
+  it('After PUT on perspective, response should not present in cache',
```
Comment (iamigo) [inline tests/enableCache/perspectives.js]: "should not be"

File: api/v1/controllers/perspectives.js [hunk 3]
```diff
@@ -160,6 +161,11 @@ module.exports = {
...
+    if (featureToggles.isFeatureEnabled('enableCachePerspective')) {
+      redisCache.del('/v1/perspectives');
```
Comment (iamigo) [inline api/v1/controllers/perspectives.js]: I guess I don't remember how perspective caching is working... are you deleting the entire perspective cache when a single perspective gets updated?

Comment (harshkothari410) [reply api/v1/controllers/perspectives.js]: Yes only one request we cache not any other request we cache in perspective. I guess it makes more sense to remove cache rather then to find that perspective in cache and update it.

**Pull request merged**